lib/image-handler.js

```diff
@@ -145,6 +145,7 @@ module.exports = function imageHandler(
         extraOpts = {
             sharpOptions: {
                 animated: true,
+                limitInputPixels: false
```

Comment:
@Steve-Mcl This was the missing piece to get your Gif with that size to be compressed properly :)